pyproj/_crs.pyx

```diff
@@ -2834,15 +2834,19 @@ cdef class _CRS(Base):
         # get list of possible matching projections
         cdef PJ_OBJ_LIST *proj_list = NULL
         cdef int *c_out_confidence_list = NULL
-        cdef int num_proj_objects = -9999
+        cdef int num_proj_objects = 0
```

Comment:
This looks like the only change needed.

Reply:
Actually, this seems safe:
```python
>>> list(range(-9999))
[]
```

Reply:
Oh, I did not know you can do this, its a bit strange though I think. Do you prefer the -9999 default or the 0? I would tend to think 0 is a bit more natural for most readers.

Reply:
It would be fine to change to 0 for readability.